elide-contrib/elide-test-helpers/src/main/java/com/yahoo/elide/contrib/testhelpers/graphql/elements/VariableValue.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.testhelpers.graphql.elements;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * {@link VariableValue} models a variable name following a {@code $} sign.
+ */
+@RequiredArgsConstructor
+public class VariableValue implements ValueWithVariable {
+
+    private static final long serialVersionUID = -7483470663100670977L;
+
+    @NonNull
+    @Getter(AccessLevel.PRIVATE)
+    private final String name;
+
+    @Override
+    public String toGraphQLSpec() {
+        // variable : '$' name;
```

Comment:
I think `'$'name` is clearer to me.

Reply:
[This is a definition](https://github.com/graphql-java/graphql-java/blob/master/src/main/antlr/GraphqlCommon.g4#L60)